sound/soc/codecs/rt715-sdw.c

```diff
@@ -7,8 +7,6 @@
  * ALC715 ASoC Codec Driver based Intel Dummy SdW codec driver
```

Comment:
@jack-cy-yu please split this in two separate commits and for each one explain why the change is needed. Also, make sure you update the 'tags'in commit message according to previous commits in this file.

Should be something like ASoC: codecs: rt715-sdev: ....

Reply:
@dbaluta aren't these already two separate commits? Or what you mean is two separate PR?

Reply:
@jack-cy-yu I think he meant to separate the removal of DEBUG from the change in return value.